packages/ses/src/error/console.js

```diff
@@ -362,12 +394,51 @@ const makeCausalConsole = (baseConsole, loggedErrorHandler) => {
   return /** @type {VirtualConsole} */ (freeze(causalConsole));
 };
 freeze(makeCausalConsole);
-export { makeCausalConsole };
 
-// /////////////////////////////////////////////////////////////////////////////
+/**
+ * @typedef {(...args: unknown[]) => void} Logger
+ */
+
+/**
+ * @param {LoggedErrorHandler} loggedErrorHandler
+ */
+export const makeCausalConsoleFromLoggerMaker = loggedErrorHandler => {
```

Comment:
I think we've tried to avoid the "make…Maker" naming. @gibson042 had a better scheme though I don't recall whether it was written up. With Exos a function that produces a `make` is called a `define`.
```suggestion
export const defineCausalConsoleFromLogger = loggedErrorHandler => {
```

Reply:
Done

Reply:
> I think we've tried to avoid the "make…Maker" naming. @gibson042 had a better scheme though I don't recall whether it was written up. With Exos a function that produces a `make` is called a `define`.

I don't remember the details, but I think that resulted in https://github.com/Agoric/agoric-sdk/pull/5234 .

Reply:
It's also used throughout the exo class making functions, and even the earlier deprecated virtual kind making functions.